db_sqlite.go

```diff
@@ -70,6 +70,15 @@ CREATE TABLE DeliveryReceipt (
 	FOREIGN KEY(network) REFERENCES Network(id),
 	UNIQUE(network, target, client)
 );
+
+CREATE TABLE ReadReceipt (
+	id INTEGER PRIMARY KEY,
+	network INTEGER NOT NULL,
+	target VARCHAR(255) NOT NULL,
+	timestamp INTEGER NOT NULL,
```

Comment:
I'd prefer to pick one of the [standard ways](https://www.sqlite.org/datatype3.html#date_and_time_datatype) to store timestamps in SQLite, so that built-in functions and third-party tools can easily parse the field. For the precision we require, `TEXT` [should be okay](https://pkg.go.dev/github.com/mattn/go-sqlite3#SQLiteTimestampFormats) (and we should be able to pass plain `time.Time` values to the driver).